intezer_sdk/api.py

```diff
@@ -219,6 +219,12 @@ def assert_any_on_premise(self):
         if self.on_premise_version:
             raise errors.UnsupportedOnPremiseVersionError('This endpoint is not available yet on on-premise')
 
+    @staticmethod
+    def is_intezer_site_available() -> bool:
+        response = requests.get('https://analyze.intezer.com/api/v2-0/is-available')
+        is_available = response.json().get('result', {}).get('Is available')
```

Comment:
Are you sure it's `Is available`, the docs says otherwise

Reply:
You are right, I got confuse with Splunk soar:
![image](https://github.com/intezer/analyze-python-sdk/assets/23573428/272ab0f0-6cb9-4769-908a-17b52318e74a)
![image](https://github.com/intezer/analyze-python-sdk/assets/23573428/911b0119-62e3-4e34-9bfc-06011e8636c5)